Neos.Neos/Classes/Fusion/Helper/BackendUserHelper.php

```diff
@@ -0,0 +1,44 @@
+<?php
+namespace Neos\Neos\Fusion\Helper;
+
+/*
+ * This file is part of the Neos.Neos package.
+ *
+ * (c) Contributors of the Neos Project - www.neos.io
+ *
+ * This package is Open Source Software. For the full copyright and license
+ * information, please view the LICENSE file which was distributed with this
+ * source code.
+ */
+
+use Neos\Eel\ProtectedContextAwareInterface;
+use Neos\Flow\Annotations as Flow;
+use Neos\Neos\Service\UserService;
+
+/**
+ * BackendUser helper for translations in the backend
+ */
+class BackendUserHelper implements ProtectedContextAwareInterface
```

Comment:
Thinking whether `BackendHelper` would be better and we could add more methods to help with hints for backend users and stuff. Wdyt?

Reply:
yes it does make sense to not limit it to the User but make it more generic.

so
`${Neos.Backend.interfaceLanguage()}`

we should also make an additional check if we are even in the backend and throw otherwise...